src/type/preload.ts

```diff
@@ -466,6 +471,7 @@ export const hotkeyActionNameSchema = z.enum([
   "元に戻す",
   "やり直す",
   "新規プロジェクト",
+  "全画面表示/ウィンドウ表示切り替え",
```

Comment:
丁寧な表記いいですね！
ちょっと長いのでこうしちゃうのどうでしょ！
```suggestion
  "全画面表示の切り替え",
```
（slackが「フルスクリーン表示の切り替え」だったので真似してみました）